cmd/storagenode/main.go

```diff
@@ -319,6 +390,42 @@ func isOperatorWalletValid(wallet string) error {
 	return nil
 }
 
+// clr clears the screen so it can be redrawn
+func clr() {
+	var clear = make(map[string]func())
+	clear["linux"] = func() {
```

Comment:
for the linux and darwin cases, you only need to printf "\033[H\033[J". that's the ANSI escape sequences for "move the cursor to the home position" and "clear the screen after the cursor".

Reply:
maybe Windows too, as I remember hearing that cmd.exe supports ANSI escape codes now.